server/dev-server.js

```diff
@@ -3,6 +3,7 @@ require('../build/check-versions')();
 
 // dependencies
 require('dotenv').config({ path: '/etc/kiva-ui-server/config.env' });
+require('dotenv').config({ path: './config.env' });
```

Comment:
Multiple requires here are fine, `/etc...` is protected on the mac so the app couldn't reach it without a lot of mucking about with permissions that might affect other things.